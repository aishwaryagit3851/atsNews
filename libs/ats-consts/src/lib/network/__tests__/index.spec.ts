import { USER_STATUS, SUCCESS_RESPONSE, ERROR_RESPONSE } from '..';

describe('ATS CONSTS for - network', () => {
  it('Match consts to snapshot', () => {
    expect(USER_STATUS).toMatchSnapshot();
    expect(SUCCESS_RESPONSE).toMatchSnapshot();
    expect(ERROR_RESPONSE).toMatchSnapshot();
  });
});
