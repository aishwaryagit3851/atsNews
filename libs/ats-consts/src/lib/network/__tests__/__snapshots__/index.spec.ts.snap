// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ATS CONSTS for - network Match consts to snapshot 1`] = `
{
  "ACTIVE": "Active",
  "BLOCKED": "Blocked",
  "PENDING": "Pending",
  "SUSPENDED": "Suspended",
}
`;

exports[`ATS CONSTS for - network Match consts to snapshot 2`] = `
{
  "data": {},
  "message": "Success",
  "success": true,
}
`;

exports[`ATS CONSTS for - network Match consts to snapshot 3`] = `
{
  "message": "Error",
  "success": true,
}
`;
